test/els_diagnostics_SUITE.erl

```diff
@@ -91,6 +92,41 @@ compiler(Config) ->
               ),
   ok.
 
+-spec elvis(config()) -> ok.
+elvis(Config) ->
+  Cwd = file:get_cwd(),
+  RootPath = ?config(root_path, Config),
+  try
+      file:set_cwd(RootPath),
+      Uri = ?config(elvis_diagnostics_uri, Config),
+      ok = els_client:did_save(Uri),
+      {Method, Params} = wait_for_notification(),
+      ?assertEqual( <<"textDocument/publishDiagnostics">>
+                  , Method),
+      ?assert(maps:is_key(uri, Params)),
+      #{uri := Uri} = Params,
+      ?assert(maps:is_key(diagnostics, Params)),
+      #{diagnostics := Diagnostics} = Params,
+      ?assertEqual(2, length(Diagnostics)),
+      Warnings = [D || #{severity := ?DIAGNOSTIC_WARNING} = D <- Diagnostics],
+      Errors   = [D || #{severity := ?DIAGNOSTIC_ERROR}   = D <- Diagnostics],
+      ?assertEqual(2, length(Warnings)),
+      ?assertEqual(0, length(Errors)),
+      [#{range := WarningRange1}
```

Comment:
Nit: missing space between `[` and `#`,  and also `,` and `#` in the next line.

Reply:
Can we configure elvis to complain about those?

Reply:
Space after commas should be the default, not sure why it wasn't detected. Spaces before `#`  wouldn't make much sense IMO.